turbinia/workers/photorec.py

```diff
@@ -66,4 +66,4 @@ def run(self, evidence, result):
     except TurbiniaException as exception:
       result.close(self, success=False, status=str(exception))
 
-    return result
+    return result
```

Comment:
There are a few of these removing trailing newline changes for unrelated files here (and in some of your other PRs).  I'm guessing that this is something that your editor is doing automatically.   It's pretty standard in unix to keep these newlines, and it's usually the editor default to do so, so just to keep the code-base consistent and also not have the extra noise in the commit history, could you potentially revert these and maybe check your editor settings?

This might be useful:
https://forum.sublimetext.com/t/make-saving-newline-at-eof-the-installation-default/9842

Reply:
That helped a lot, thanks!